web/src/components/molecules/core/SearchBar.tsx

```diff
@@ -1,3 +1,16 @@
+<<<<<<< HEAD
+import * as React from "react";
+import { styled } from "@material-ui/core/styles";
+import { ClearIcon, SearchButton, Container } from "../../atoms";
+import { InputField } from "../../molecules";
+import {
+  OnChange,
+  InputChangeElement,
+  OnClick,
+  ButtonClickElement,
+} from "../../../types";
+import { flexStyle, flexRowStyle } from "../../../styles";
+=======
```

Comment:
another conflicted file

Reply:
Oops, these are fixed in #95, but will fix them here, too!